scanspec/service.py

```diff
@@ -88,19 +88,33 @@ def validate_spec(spec: Spec) -> Any:
 
 
 # Returns a full list of points for each axis in the scan
-# TODO adjust to return a reduced set of scanPoints
-def get_points(spec: Spec) -> PointsRequest:
-    """ A query that takes a Spec and calculates the points present in the scan
+# TODO Update max_points with a more sophisticated method of reducing scan points
+def get_points(spec: Spec, max_points: Optional[int] = 200000) -> PointsRequest:
+    """A query that takes a Spec and calculates the points present in the scan
     (for each axis) plus some metadata about the points.
+
+    Arguements:
+            [spec]: [The specification of the scan]
+            [max_points]: [The maximum number of points the user wishes to receive]
+
+    Returns:
+        [PointsRequest]: [A dataclass containing information about the scan points
+                            present in the spec]
     """
     dims = spec.create_dimensions()  # Grab dimensions from spec
     path = Path(dims)  # Convert to a path
     num_points = len(path)  # Capture the length of the path
```

Comment:
Could this be total_points or something now to avoid any ambiguity? Maybe that also requires a name change in PointsRequest? And also use this in line 112 and 113

Reply:
Currently this is actually the total number of frames returned so, I think total_frames would be better suited here

Reply:
On that same line of thinking, isn't max_points technically max_frames as well?